nvtabular/inference/triton/workflow_model.py

```diff
@@ -50,15 +42,18 @@
 
 
 class TritonPythonModel:
+    """
+    Triton model used for running NVT Workflows
+    """
+
     def initialize(self, args):
         # Arg parsing
-        workflow_path = os.path.join(
-            args["model_repository"], str(args["model_version"]), "workflow"
-        )
+        repository_path = pathlib.Path(args["model_repository"]).parent.parent
```

Comment:
the change to what `model_repository` represents might be something to do with this PR? https://github.com/triton-inference-server/python_backend/pull/138

Based on the name `model_repository`, if it's intended to be the same as the tritonserver `model_repository` I'd expect this to match the directory of the model repository passed to triton (the one with all the model directories in it). However even before the changes in release 22.06, it seems like it was set to "{model_repository}/{model_name}".

Reply:
Yeah, I am confused by what it's intended to convey. This seems to work, but it looks pretty weird to me.